### ISSUE TITLE
[MHV Medications / Accessibility] Some content in accordion panels not included in reading order

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#125268]
### 📝 User story

As a screen reader user, I want content that is available visually to be included in reading order, so that I don't miss out on informational content that is available visually.

### 📝 Problem statement

Information related to images is hidden from screen reader users.

### 🌌 Supporting screenshots or videos

<img width="547" height="618" alt="Screenshot of accordion component with an overlay showing content that is hidden from screen readers" src="https://github.com/user-attachments/assets/3d73eb43-2f84-4e68-a149-07e8dc31257b" />

Details about images are hidden using the `aria-hidden `attribute 

```
<h4 … aria-hidden="true">Image</h4>
<div … aria-hidden="true"><p …>Image not available</p></div>
```

### 👣 Reproduction steps

1. Navigate to the [Medications detail page](https://staging.va.gov/my-health/medications/prescription/24654491) (username: (mhvmark.test@id.me).
2. Use the ARC Toolkit browser extension to show elements with the `aria-hidden` attribute.
3. Use a screen reader in reading mode to read through accordion content.
4. Notice that the details related to images are not announced by the screen reader. 

### 🔗 Point of contact

@sarahhortonadhoc

### ⚠️ Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

#### Accessibility Standards
- [Guideline 1.3 Adaptable: Create content that can be presented in different ways (for example simpler layout) without losing information or structure.](https://www.w3.org/WAI/WCAG22/Understanding/adaptable)
  - [1.3.2 Meaningful Sequence (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/meaningful-sequence.html)

#### Experience Standards
- [Comprehension: User easily understands the product](https://design.va.gov/about/experience-standards/#category-comprehension)
  - [User can perceive all meaningful elements](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
 
### 𝌣 Proposed solution or next steps

Remove the `aria-hidden` attribute from the image details.

### ✅ Acceptance criteria

- [ ] The system provides the image details in the reading order.
- [ ] The screen reader user can access the image details.

### ✅ Possible tasks

- [ ] Review, refine, and acknowledge feedback.
- [ ] Update ticket information (title, assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close the ticket after reviewing documented decisions / validating the fix.

**Comment by ckeif-iat** [comment]
It looks like this may have already been addressed in [this commit](https://github.com/department-of-veterans-affairs/vets-website/commit/ec57cb59e74a60f5f0e3ea380a0fac3bd613c84c).

While testing locally on `main` it looks like it works as expected:

https://github.com/user-attachments/assets/837a9f12-9b3d-43d4-a7dd-33c43bd0735f

**Comment by sarahhortonadhoc** [comment]
@ckeif-iat Just tested on staging with VO and "Image" and "Image not available" are announced. @amycashbaugh confirming that you addressed this issue in [Commit ec57cb5](https://github.com/department-of-veterans-affairs/vets-website/commit/ec57cb59e74a60f5f0e3ea380a0fac3bd613c84c)?
